Packs/Akamai_SIEM/ReleaseNotes/1_1_9.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### Akamai WAF SIEM
+***breaking Changes:*** Added limitations to **Fetch limit** and **page size** parameters due to Restrictions from the platform. The maximum allowed amount is 45k. Note that this change will not fail the execution, but if higher rates are configured, the fetch will use 45k.
```

Comment:
i dont think we need to release it as breaking change as it should only improve performance. Add a jnote about how if the maximum is brought the next run will start immediately (add this to the docs above too)